app/Models/Matter.php

```diff
@@ -164,6 +190,15 @@ public function annuityAgents()
             ->wherePivot('role', 'ANN');
     }
 
+    public function sharedAnnuityAgents()
```

Comment:
There must not be more than one, yet I don't recall this is checked. Adding more than one annuity agent, like more than one client, could have wrong side effects, so we'd need to design some safeguards (another project).